hack/helm/test.sh

```diff
@@ -29,11 +32,14 @@ check() {
     helm lint --strict .
```

Comment:
Thanks for catching this.  Would we want to also fail if linting fails?

```suggestion
    helm lint --strict . || STATUS=1
```

Reply:
Yes indeed. Looking back, I wonder why we don't just `set -e` to fail on the first error?

Reply:
💯 .  We should likely just do that, and ` || true` anything that isn't fatal.

Reply:
Was the original intention precisely not to stop at the first error and reveal them all in a single execution? I'm not sure it's worth it because the script runs very fast.